src/main/java/com/alrex/parcool/common/action/impl/Dodge.java

```diff
@@ -171,32 +174,23 @@ public boolean canContinue(Player player, Parkourability parkourability, IStamin
 	@Override
 	public void onStartInLocalClient(Player player, Parkourability parkourability, IStamina stamina, ByteBuffer startData) {
 		dodgeDirection = DodgeDirection.values()[startData.getInt()];
+		var dodgeVec = getDodgeVec(player, startData);
+		dodgeDirection = AdditionalMods.betterThirdPerson().handleCustomCameraRotationForDodge(dodgeDirection);
+		dodgeDirection = AdditionalMods.shoulderSurfing().handleCustomCameraRotationForDodge(dodgeDirection);
```

Comment:
I think deciding of `dodgeDirection` should be done before start (concretely in `canStart`). To sync the value in evely client and server.

Reply:
I needed to move it here because I was losing the value of dodgeVec passed using startData in some cases, I created getDodgeVec just because of that. But that was unnecessary. All the troubles I had were because I passed y instead of z and didn't notice. I'll fix it right away